cypress/e2e/oob_invitation_usecase.cy.ts

```diff
@@ -0,0 +1,142 @@
+const API_URL = Cypress.env('apiUrl')
+const TEST_AGENT_URL = 'http://localhost:9000'
+
+describe('UseCase Page', () => {
+  it('successfully completes school use case', () => {
```

Comment:
Can you change the naming here to make it more related to what you are testing?

We should also include tests for the credentialVersion.

Reply:
On it